apps/tsconfig.json

```diff
@@ -15,6 +15,18 @@
       "@defichain-apps/libs/*": [
         "libs/*/src"
       ],
+      "@defichain/jellyfish-api-core/dist/*": [
```

Comment:
would this work? (sometimes the mapper just weird)
```
"@defichain/*/dist/*": [
  "../packages/*/src/*"
]
```

Reply:
typescript only allows for 1 `*` wildcard unfortunately. If you try with multiple you will get a ts error. ☹️